lib/internal/process/per_thread.js

```diff
@@ -258,6 +261,19 @@ function wrapProcessMethods(binding) {
     };
   }
 
+  /**
+   * Loads the `.env` file to process.env.
+   * @param {string | URL | Buffer | undefined} path
+   */
+  function loadEnvFile(path = undefined) {
+    if (path != null) {
+      path = getValidatedPath(path);
+      _loadEnvFile(toNamespacedPath(path));
+    } else {
+      _loadEnvFile();
+    }
```

Comment:
```suggestion
    if (path === undefined) {
      _loadEnvFile();
    } else {
      path = getValidatedPath(path);
      _loadEnvFile(toNamespacedPath(path));
    }
```

Generally the default/affirmative condition goes first. We also don’t want to do `!= null` and then values like `true` are accepted.

Reply:
> We also don’t want to do `!= null` and then values like `true` are accepted.

What are you talking about?  `v != null` is equivalent to `v !== null && v !== undefined`, we use it everywhere, and both treat `true` the same.

Reply:
And `true` is not one of the accepted inputs. It should fall through to the second block where it will error.

Reply:
Your suggestion changes nothing for `true`, not sure why your mentioning it. FWIW I prefer the nullish check (i.e. as is).

Reply:
We want to make sure `null` and `undefined` value should throw error as well. So, I don't see any problem with the current approach. Also, with your recommendation, if you pass `null` value, it will get executed by calling `getValidatedPath()` and it will throw different errors for `null` and `undefined`

Reply:
That’s fine, I was just thinking it was clearer to make explicit that the default case is first and would only happen explicitly for `undefined`. What I meant about `true` was that in my version it would fall through to the case with `getValidatedPath` and error there, like it does in the original.